ami/flowchart/library/Operators.py

```diff
@@ -492,12 +492,12 @@ def to_operation(self, **kwargs):
             expr = self.values['operation']
 
             # sympy doesn't like symbols name likes Sum.0.Out, need to remove dots.
-            for arg in self.input_vars().values():
+            for arg in list(self.input_vars().values())[::-1]:  # Reverse order so "In" does not messes up with the replacement if "In.1"
```

Comment:
Typical example of where this was a problem:
![image](https://github.com/user-attachments/assets/fbfbc536-2944-4758-8469-cbe17e372cca)

In this case, it would first try to replace MeanVsScan.0.Counts, hereby replacing part of MeanVsScan.0.Counts.1

By reverting the order, we prevent the first input (which does not have an index .0) to replace the string everwhere and mess up with the other variables